### ISSUE TITLE
Inquiry About Column Drop Parameter Settings

### ISSUE THREAD

**Issue by JoshuaHugh** [drumpt/AdapTable#1]
Dear Authors,

Thank you for your valuable work. I have some questions about the Column Drop implementation:

Based on this repository, /conf/config.yaml and /conf/ours_mlp.yaml show:
     
```yaml
     mae_imputation_method: "emd"
     pretrain_mask_ratio: 0.75  
```

However, this paper mentions that in column drop, the random mask $m_j$ is set to $\text{Bernoulli}(0.2)$. Is there any conflict between the code and what is described in the paper?

Moreover, "emd" needs to use the training data, is this a violation of the requirement of not being able to access the training data in fully test-time settings?

I hope you can answer. Thank you for your time!

**Comment by drumpt** [comment]
Dear Joshua Hugh,

Thank you very much for your interest in our research project and for taking the time to review our code.
The configurations you found with pretrain_mask_ratio: 0.75 are legacy settings from development experiments, not the final Column Drop implementation. As the text explains, these were only for pretraining POC tests. The actual Column Drop method uses Bernoulli(0.2) masking, as stated in the paper.

Best wishes,
Changhun Kim

**Comment by JoshuaHugh** [comment]
Dear Authors,

Thank you for your reply. I also wonder which kind of mae_imputation_method you set for Column Drop in the paper because as far as I know, models like ft-transformer can't automatically handle NAN.

I look forward to your reply.

**Comment by drumpt** [comment]
Dear Joshua Hugh,

We simply used zero imputation.
Please refer to our code [https://github.com/drumpt/AdapTable/blob/main/data/dataset.py#L54-L56](https://github.com/drumpt/AdapTable/blob/main/data/dataset.py#L54-L56).

Thank you.

**Comment by JoshuaHugh** [comment]
Dear Authors,

Thank you so much for your reply :-) .

**Comment by drumpt** [comment]
Dear Joshua Hugh,

Thank you for your interest in our research.
We wish you the best of luck with your research as well.

Thank you.
